cloud/main.js

```diff
@@ -17,6 +17,17 @@ Parse.Cloud.define("hello", function(request, response) {
     response.success(getAliasName());
 });
 
+Parse.Cloud.define("replayMessages", function(request, response) {
+    var requiredParams = ["chatRoomId", "subkey", "count"];
```

Comment:
shouldnt this have some sort of timetoken param too?

See https://www.pubnub.com/docs/web-javascript/storage-and-history#paging